SpeechResponder/ConfigurationWindow.xaml.cs

```diff
@@ -276,8 +275,15 @@ private void copyPersonalityClicked(object sender, RoutedEventArgs e)
                 string PersonalityName = window.PersonalityName?.Trim();
                 string PersonalityDescription = window.PersonalityDescription?.Trim();
                 Personality newPersonality = Personality.Copy(PersonalityName, PersonalityDescription);
-                Personalities.Add(newPersonality);
-                Personality = newPersonality;
+                if (newPersonality == null)
+                {
+                    var _ = MessageBox.Show(window, Properties.SpeechResponder.name_taken, "", MessageBoxButton.OK, MessageBoxImage.Asterisk);
```

Comment:
Use a try-catch statement here and show the message box on an `InvalidOperationException`?